packages/plans-grid/src/plans-table/style.scss

```diff
@@ -241,10 +241,30 @@
 	// stylelint-disable-next-line scales/radii
 	border-radius: 4px;
 	margin-top: 15px;
-	border: 1px solid var( --studio-gray-5 );
+	
+	&:not( .is-popular ) {
+		border: 1px solid var( --studio-gray-5 );
+	}
+
+	&.is-selected {
+		// popular selected items don't need a blue border
+		&:not( .is-popular ) {
+			border: 1px solid var( --studio-blue-40 );
+			color: var( --studio-blue-40 );
```

Comment:
I've tested so far only in Focused Launch and I see different colors for popular plan button (`--wp-admin-theme-color` seems to be the default bg color for primary button) and the selected plan button. 

So this AC is not ✅ yet:

> The 2 changes above should also override any other button styles (including the primary button used in the "popular" plan)